### DISCUSSION TITLE
Dev and Stable version are not recognized by ACARS addon

### DISCUSSION THREAD

**Discussion by supdim** [flybywiresim/aircraft#5217] [Help] [answered]
I'm flying for a virtual airline and therefore I'm using an ACARS using FSUIPC. The ACARS only detects the Experimental version. I have absolutly no idea, why the Dev or the stable version were not recognized. Any ideas?

**Comment by tracernz** [comment, marked as answer]
If it is open-source then maybe somebody could check, otherwise the only option is the ACARS developer.

**Reply by supdim** [reply]
The crazy thing is, other pilots in the VA using the FBW too and they have no problems with the ACARS at any version.

**Comment by Ermy64** [comment]
Hello everyone, I tried the stable 6.3 and my ACARS does not give problems with the parking brake, instead with the version 7.3 not yet official the ACARS detects me that the parking brake is not engaged. I am aware that version 7.3 is not yet stable, and I hope it will not happen when the stable version comes out.All these problems with the parking brake started with the introduction of the hydraulic panel. Thanks for your time.

**Reply by tracernz** [reply]
> All these problems with the parking brake started with the introduction of the hydraulic panel.

Indeed, that is when the sim's unrealistic parking brake was replaced with a realistic simulation using different simvars (as we cannot write the sim's ones).

**Comment by mrugoVe** [comment]
it also happens to me two acars systems do not recognize the parking brake that after last update

**Comment by mrugoVe** [comment]
it also happens to me two acars systems do not recognize the parking brake that after last update
